odl/contrib/shearlab/examples/Benchmarks.ipynb

```diff
@@ -0,0 +1,526 @@
+{
```

Comment:
Great to have benchmarks!

Just two remarks on this:
- Please import the functions from the module instead of copying the code into the notebook:
   ```python
   from odl.contrib.shearlab.shearlab_operator import (
       <function names go here>)
   ```
- If you like, use `odl.util.Timer` instead of `tic` and `toc`, the output will be a bit more informative:
   ```python
   >>> with odl.util.Timer('Time for sleep(1.5)'):
   ...     sleep(1.5)
   Time for sleep(1.5) :      1.513
   ```